node/src/Pos/Lrc/Worker.hs

```diff
@@ -197,7 +197,7 @@ lrcDo epoch consumers tip = tip <$ do
     whileAfterCrucial b = getEpochOrSlot b > crucial
     crucial = EpochOrSlot $ Right $ crucialSlot epoch
     withBlocksRolledBack blunds =
-        bracket_ (rollbackBlocksUnsafe blunds)
+        bracket_ (rollbackBlocksUnsafe (BypassSecurityCheck True) blunds)
```

Comment:
The reason is: we might call LRC rollback during `applyWithRollback`, which himself might rollback blocks and both these rollbacks (LRC and `applyWithRollback`) are summarized and give too deeper rollback.

rollback in the `applyWithRollback` might not be deeper than `k`, but LRC rollback we create artificially. To avoid deep artificial rollback we ignore invariant in the LRC.